pydatalab/pydatalab/models/starting_materials.py

```diff
@@ -31,7 +31,9 @@ class StartingMaterial(Item):
 
     full_percent: Optional[str] = Field(alias="Full %")
 
-    name: str = Field(alias="Container Name", description="name of the chemical")
+    GHS_codes: Optional[str] = Field(alias="GHS H-codes")
```

Comment:
Is a simple string enough here? Should it at the very least be a list of strings? In terms of making this "actionable" it would be good to have a GHS  H-code type/enum so that we can show proper warnings in the future.

This would however involve some extra validation and a better feedback loop between UI and server for this...

Reply:
Also a description would be nice, with reference to https://pubchem.ncbi.nlm.nih.gov/ghs/, perhaps a couple of examples of what this field could look like in an `examples` field

Reply:
I agree, but have left this as a string for now because we don't have an easy way of showing lists in the frontend at the moment, except maybe a multiselect. I would like use to do things like auomatically show the pictograms and automatically populate the hazards based on CAS in the future, but in my opinion its better to have a string field for now than nothing. I will add a description, though.

Reply:
issue #657

Reply:
That would work for visualizing, but not for entering the codes in the first place through the UI

Reply:
It will also be an issue in cheminventory xlsx imports, which appear to be strings of space-separated codes (e.g. "H260 H314")